Python/ds/Queue.py

```diff
@@ -0,0 +1,32 @@
+# Stack implementation using List in Python
+
+class Queue :
+  def __init__(self):
+    self.items =[]
+    
+  def print_queue(self):
+    print "\nQueue: ", self.items
```

Comment:
As most of the python codes are available in python 3, I would recommend to use print() instead of print.

Reply:
Sure, I'll do that